config/routes.rb

```diff
@@ -3,9 +3,16 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   root to: 'movies#index'
+
+
+  resources :questions, only: [:index, :show, :create, :edit, :update]
+  resources :aws_texts
```

Comment:
`resources :aws_texts`が2つありますね。あと，一番下の3行とまとめてこう書くべきですね。

```suggestion
  resources :questions, only: [:index, :show, :create, :edit, :update] do
    resource :solutions, only: [:create]
  end
```